clis/interderpcli/src/main.rs

```diff
@@ -179,7 +184,10 @@ async fn ping_each_other(
         }
     })
     .await
-    .context("Second client did not receive the ping message from the first client in 10s")??;
+    .context(format!(
+        "First client did not receive the ping message from the second client in {:?}",
```

Comment:
so is it first or second? :smile:

Reply:
Omg, defeated by copy-pasting code... thanks for spotting this! :)